kie-server-parent/kie-server-controller/kie-server-controller-client/src/main/java/org/kie/server/controller/client/rest/RestKieServerControllerClient.java

```diff
@@ -295,12 +316,22 @@ private void makeDeleteRequest(String uri) {
     }
 
     private <T> T makePostRequestAndCreateCustomResponse(String uri, Object bodyObject, Class<T> resultType) {
+        return makePostRequestAndCreateCustomResponse(uri, bodyObject, resultType, new HashMap<>());
+    }
+
+    private <T> T makePostRequestAndCreateCustomResponse(String uri, Object bodyObject, Class<T> resultType, Map<String, String> headers) {
```

Comment:
Are the headers taken from anywhere? Only usage of this private method is right above, which supplies empty map.

Reply:
@sutaakar touche! that's was a change I made. Rolling back